opts/options.go

```diff
@@ -66,6 +68,9 @@ var Defaults = func(o *Options) error {
 	o.RoutingTable.AutoRefresh = true
 	o.MaxRecordAge = time.Hour * 36
 
```

Comment:
Do we not have a default for the number of Disjoint Paths because we need to set it to K/2 ?

Reply:
Exactly